engine/src/main/java/org/pentaho/di/trans/steps/pentahoreporting/PentahoReportingOutputMeta.java

```diff
@@ -138,6 +146,10 @@ private void readData( Node stepnode ) throws KettleXMLException {
     try {
       inputFileField = XMLHandler.getTagValue( stepnode, "input_file_field" );
       outputFileField = XMLHandler.getTagValue( stepnode, "output_file_field" );
+      inputFile = XMLHandler.getTagValue( stepnode, "input_file" );
```

Comment:
Just to be sure, what happens if we read an old transformation that does not have these tags ?
would we be able to read and use the transformation?

Reply:
If old transformations don't have that fields is not a problem because will be read as null and will follow the current behavior of no defined fields. The only important field is the use_values_from_fields that should be true as default for old transformations and that is covered by current solution.